docs/basics/1d_hist.rst

```diff
@@ -269,7 +269,7 @@ The examples below show how to create a histogram with a category axis in ``boos
 
 .. note::
     When we create the histograms in the examples below, we use the argument ``storage=bh.storage.Weight()``.
-    This is because functions in ``plothist`` assume histograms with a storage supporting weighted data in which the variance of each bin is tracked.
+    This is because functions in ``plothist`` assume histograms with a storage supporting weighted data in which the variance of each bin is tracked (more details `here <https://boost-histogram.readthedocs.io/en/latest/api/boost_histogram.html#boost_histogram._internal.hist.Histogram.variances>`_).
```

Comment:
Why not this https://boost-histogram.readthedocs.io/en/latest/user-guide/storage.html#weight ?